code/modules/power/tesla/coil.dm

```diff
@@ -12,6 +12,11 @@
 	var/last_zap = 0
 	var/datum/wires/tesla_coil/wires = null
 
+	// Executing a traitor caught releasing tesla was never this fun!
+	can_buckle = TRUE
+	buckle_lying = FALSE
+	buckle_requires_restraints = TRUE
```

Comment:
This means you can't buckle someone here unless they're cuffed. Intended?

Reply:
@Kyep You can be cuffed to any atmospherics pipes, including heat exchangers, only if you are restrained.

Reply:
I did not know that. I've never seen someone actually get buckled to an atmos pipe. I don't think the general playerbase are aware that this feature exists. If you want to actually see people get buckled to tesla coils, I'd suggest letting them be buckled regardless of whether they're restrained or not. Just a suggestion though.

Reply:
well the idea is you are bukcling them to the thing with the retraints so.